### DISCUSSION TITLE
issue when using react-native-canvas to display chart data

### DISCUSSION THREAD

**Discussion by chj-damon** [iddan/react-native-canvas#249] [Q&A]
- [x] I have searched the [issues](https://github.com/antvis/f2/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/chj-damon/F2-expo-reproduce](https://github.com/chj-damon/F2-expo-reproduce)

### Steps to reproduce
yarn
yarn ios

### What is expected?
the chart would display when the app starts.

### What is actually happening?
the chart didn't display at the very first time, unless some changes were made in App.tsx file.

| Environment | Info |
|---|---|
| OS | MacOS Majave |
| Environment | Expo v36 |

---

**Comment by iddan** [comment]
Thank you for the detailed report.
Which changes were made?

**Comment by chj-damon** [comment]
for example, modify the text in `<Text></Text>`. then the chart will display.

**Comment by bisk8s** [comment]
Same issue here.
But for me forcing an update won't solve it.

It only shows when a change is made leaving the emulator with the view on the screen.


![Gravação de Tela 2020-02-10 às 17 18 18](https://user-images.githubusercontent.com/5271243/74186657-db86a700-4bff-11ea-8eae-3406316cd6ba.gif)
